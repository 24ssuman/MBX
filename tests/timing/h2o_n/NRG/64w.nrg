SYSTEM h2o
MOLECULE
MONOMER h2o
O  0.025016  -0.22529  -0.78734
H  0.18573  -1.1486  -0.68604
H  0.88884  0.22847  -0.79632
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -1.9846  -0.017433  0.86606
H  -2.3656  0.83031  1.0299
H  -1.1846  0.19525  0.25791
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -0.14754  -1.036  2.4294
H  -0.16774  -2.025  2.4324
H  -0.8704  -0.59255  2.0189
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  2.4639  -0.030955  1.0414
H  1.6496  -0.4718  1.2655
H  2.9399  0.006554  1.857
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  2.1591  1.3705  -1.0761
H  2.5991  1.203  -1.8935
H  2.5096  0.95004  -0.30459
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  0.02805  -3.0207  -0.52713
H  -0.15705  -2.9941  0.45757
H  -0.83422  -3.1244  -0.94425
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -0.26348  3.1069  -0.7793
H  0.588  2.5735  -0.79602
H  -0.44978  3.3069  0.12581
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -1.2585  -0.16771  -3.2208
H  -0.39567  -0.6098  -3.3878
H  -1.2931  0.01995  -2.2203
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -2.3492  1.9601  -1.8906
H  -2.3918  2.0144  -2.828
H  -1.4647  2.2943  -1.6157
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -2.2285  -2.3753  -1.4532
H  -2.1394  -1.7775  -2.1818
H  -2.7642  -3.161  -1.8199
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -0.70486  3.4958  1.9828
H  -0.46944  4.098  2.7246
H  -1.4741  3.0103  2.2946
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -3.1104  -2.1271  2.0618
H  -3.4379  -2.8617  1.4922
H  -2.696  -1.4769  1.5435
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  1.0838  1.3423  3.8933
H  0.75549  0.63605  3.3046
H  1.3731  1.9932  3.2496
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  3.9989  -1.4052  -0.51823
H  3.3388  -0.97157  -0.059856
H  3.6231  -1.9586  -1.2413
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -3.3163  1.8319  2.3404
H  -3.9762  2.5071  2.6495
H  -3.5909  1.0589  2.824
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -0.28456  -3.7131  2.3739
H  0.53801  -4.1758  2.5047
H  -0.94809  -4.119  2.8712
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  2.454  0.35652  -3.666
H  3.4182  0.19294  -3.6829
H  2.1198  -0.50093  -3.7729
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  1.1827  -1.9085  -3.9854
H  0.88191  -1.8001  -4.9027
H  1.1864  -2.8801  -3.758
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -4.6414  0.10763  -1.3541
H  -5.3367  0.78005  -1.3387
H  -3.711  0.42449  -1.4064
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  2.1504  3.8058  2.674
H  1.4843  4.2835  3.1639
H  2.1124  4.0737  1.7092
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  4.2576  0.10163  2.8878
H  5.1036  -0.29575  3.1117
H  4.2327  0.96657  3.3234
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  2.6325  -4.5954  -0.60245
H  1.9358  -4.0853  -1.0393
H  2.4618  -4.4153  0.3083
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  2.5729  4.7975  0.32115
H  3.3201  4.7437  -0.27625
H  1.913  5.3161  -0.14728
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  0.22959  4.7876  -2.8747
H  -0.2049  4.553  -3.689
H  -0.037492  4.1668  -2.2008
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -2.3743  1.4815  -4.9305
H  -2.5541  1.2351  -5.9139
H  -1.9708  0.71471  -4.5176
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -0.86852  1.9612  5.4003
H  0.027679  1.7141  5.0598
H  -0.87458  2.032  6.4096
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -3.1861  0.44744  4.9613
H  -2.9341  -0.48801  4.7796
H  -2.3942  0.93405  5.0882
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  1.2828  1.946  -5.3951
H  1.7024  1.5812  -4.5727
H  0.82984  2.79  -5.2093
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -4.781  3.5037  -0.84801
H  -4.7647  3.6976  0.080095
H  -3.9603  2.9727  -1.042
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  5.7197  0.76855  -0.86736
H  5.134  0.005226  -0.85726
H  6.6364  0.51942  -0.70701
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  2.3456  -4.8632  2.2753
H  2.6179  -4.6409  3.1415
H  2.5069  -5.8008  2.2174
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  3.9549  -3.3186  -2.764
H  3.1426  -3.3581  -3.3089
H  4.6365  -3.0605  -3.3756
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -5.6098  -2.2969  -0.76869
H  -5.2283  -2.8768  -0.12093
H  -5.1248  -1.4299  -0.88064
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -3.4145  -2.1691  -4.548
H  -3.4922  -2.8022  -3.8008
H  -2.8429  -1.4519  -4.2509
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -5.6058  -0.93334  2.4338
H  -4.7663  -1.4416  2.3752
H  -6.1783  -1.4286  1.8355
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  1.6248  -4.5532  -3.6716
H  2.1633  -5.053  -4.3058
H  1.1067  -5.2253  -3.2379
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -1.1012  -5.9868  0.58375
H  -0.80758  -5.1298  0.92625
H  -0.61836  -6.102  -0.24071
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  3.2226  -2.2604  4.6917
H  3.1607  -1.698  3.9022
H  2.4283  -2.8291  4.8025
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -3.5885  -4.3174  -2.7083
H  -3.0609  -5.0971  -2.9484
H  -4.4165  -4.549  -2.2641
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  0.37617  -0.94717  -6.2587
H  0.47464  0.032602  -6.1327
H  -0.31565  -1.1337  -6.8644
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -3.3435  -2.4458  4.8611
H  -4.0943  -2.8828  5.1961
H  -3.3089  -2.5534  3.8656
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  4.5401  3.0258  3.3133
H  3.6414  3.4361  3.2745
H  5.2676  3.4906  3.7412
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  0.442  6.5065  -0.42973
H  -0.30517  6.7351  0.14278
H  0.1946  6.3636  -1.2619
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  0.17832  5.3152  3.973
H  -0.098588  5.124  4.8529
H  0.85767  5.9528  4.1582
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  5.122  -0.05318  -4.1003
H  5.0868  -0.80784  -4.7211
H  5.9532  -0.1056  -3.6204
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -0.92175  4.09  -5.2192
H  -1.519  3.3056  -5.3117
H  -0.74944  4.4825  -6.0792
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  5.8736  2.8238  0.96999
H  5.6989  2.0247  0.41343
H  5.2837  2.9865  1.7227
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  0.86755  -6.4325  -1.336
H  0.98908  -7.3868  -1.5718
H  1.6823  -6.2515  -0.78433
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -1.7554  -5.0603  4.0933
H  -2.6001  -5.4948  4.0165
H  -1.1657  -5.8153  4.3274
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  1.3233  -4.175  5.0812
H  0.57445  -3.7751  5.5911
H  1.0526  -5.0744  4.8442
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -5.194  -4.0969  1.5937
H  -4.8186  -4.7251  2.2191
H  -6.0845  -3.7678  1.9433
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -2.4838  4.0978  4.8961
H  -1.8861  3.3183  4.8493
H  -3.1148  4.0916  4.137
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -5.7121  -0.61059  -3.8317
H  -5.0116  -1.055  -4.3763
H  -5.2781  -0.41781  -2.9933
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -1.1173  -2.5517  6.384
H  -1.9112  -2.3905  5.7707
H  -0.81618  -1.6656  6.6987
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  4.9255  4.8645  -0.82847
H  4.7484  5.7059  -1.2732
H  5.2774  4.1963  -1.4602
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -6.8047  1.7385  1.1496
H  -6.4156  0.87887  1.1733
H  -6.9774  1.8165  0.20427
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  5.5458  -3.8852  1.6168
H  4.7404  -4.4641  1.6071
H  6.1212  -4.3096  1.0157
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  2.7543  0.53552  6.5045
H  3.4386  -0.11341  6.7964
H  2.5978  0.41173  5.5991
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -4.5669  4.54  3.2512
H  -4.7892  4.9339  2.4345
H  -5.4184  4.5653  3.7425
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -1.4325  -6.305  -3.107
H  -0.55279  -6.2586  -2.6553
H  -1.0943  -6.7596  -3.9101
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -6.8993  1.4965  -1.7684
H  -6.7927  2.2969  -2.3284
H  -7.8109  1.2873  -1.9191
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  -1.1646  6.992  1.6825
H  -1.072  6.3745  2.4287
H  -2.0546  7.392  1.7644
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  6.3982  -1.4997  2.9137
H  6.3399  -2.2535  2.3344
H  6.9498  -0.80279  2.5997
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  6.8999  -2.2249  -0.47662
H  7.1045  -3.1449  -0.66564
H  6.0911  -2.249  -0.011097
ENDMON
ENDMOL
ENDSYS
