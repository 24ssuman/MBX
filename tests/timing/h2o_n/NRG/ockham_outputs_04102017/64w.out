Energies without gradients:
system[....0]=     3.9698125880e+02    kcal/mol
Energies with gradients:
system[....0]=     3.9698125880e+02    kcal/mol
Energy:(nograd) 101 milliseconds
Energy:(grad) 202 milliseconds
1B:(nograd) 0 milliseconds
1B:(grad) 0 milliseconds
2B:(nograd) 11 milliseconds
2B:(grad) 23 milliseconds
3B:(nograd) 89 milliseconds
3B:(grad) 178 milliseconds
Get and set 2b clusters: 0 milliseconds
Get and set 3b clusters: 0 milliseconds
